### ISSUE TITLE
EPC Import

### ISSUE THREAD

**Issue by alexander-griffen** [mysociety/local-intelligence-hub#126]
Energy efficiency of housing (average EPC) data.
[Possibly found here](https://www.cse.org.uk/projects/view/1259#Energy_Performance_Certificate_data), although this is quite old data.
Alternatively, [this dataset](https://www.ons.gov.uk/peoplepopulationandcommunity/housing/datasets/estimatedproportionofdwellingswithanenergyperformancecertificatebymainfueltypeormethodofheatingusedincentralheating) from the ONS has more up-to-date data, but only for rented properties.

More searching may find a better data source.

**Comment by zarino** [comment]
@ajparsons You recently looked into EPC data – do you know of a good source for whole-UK owned+rented EPC data? (If such a thing exists!)

**Comment by ajparsons** [comment]
No, it's known not to - needs to be reconstructed from separate e+w and scottish data, plus an FOI for NI data.

**Comment by zarino** [comment]
🤦

**Comment by ajparsons** [comment]
This exists for E+W though - https://www.owenboswarva.com/blog/post-hou7.htm

**Comment by zarino** [comment]
The House of Commons Library recently published a dashboard of this (England and Wales only) EPC data: https://commonslibrary.parliament.uk/constituency-data-energy-efficiency/
